src/script/script.cpp

```diff
@@ -156,6 +156,25 @@ std::string GetOpName(opcodetype opcode)
     case OP_INSPECTOUTPUT          : return "OP_INSPECTOUTPUT";
     case OP_INSPECTTX              : return "OP_INSPECTTX";
 
+    // 64 bit LE arithmatic opcodes
```

Comment:
spelling: `arithmetic` (I had to look this one up :)). Appears in a few places and also in the commit message.